crates/holochain/src/core/ribosome/host_fn/get_agent_activity.rs

```diff
@@ -16,6 +18,10 @@ pub fn get_agent_activity(
 ) -> Result<AgentActivity, RuntimeError> {
     match HostFnAccess::from(&call_context.host_context()) {
         HostFnAccess {
+            write_workspace: Permission::Allow,
+            ..
+        }
+        | HostFnAccess {
```

Comment:
Shouldn't this require both read & write permission, not either/or?

Reply:
updated https://github.com/holochain/holochain/pull/5377/commits/9bac3eb19e685d57780aad22b22fb567f6da5b4a